### PR TITLE
Feature/interface details weather

### PR DESCRIPTION
**Pull request by pedrohsampaioo**
## Interfaces
- **Atmosphere** (day and night)
- **Clear** (day and night)
- **Clounds** (day and night)
- **Drizzle** (day and night)
- **Rain** (day and night)
- **Snow** (day and night)
- **Thunderstorm** (day and night)

### CODE REVIEW

File: app/lib/app/modules/home/home_page.dart [hunk 1]
```diff
@@ -12,16 +15,119 @@ class HomePage extends StatefulWidget {
...
+          Center(
+            child: Padding(
+              padding: const EdgeInsets.all(16),
+              child: Text("$weatherMain selecionado!!"),
+            ),
+          ),
+        ],
       ),
+      floatingActionButton: FloatingActionButton(
+        child: Text("ABRIR"),
```
Comment (marceloamadeu) [inline app/lib/app/modules/home/home_page.dart]: Hi @pedrohsampaioo.

Going to change to "**OPEN**"?

Tks

File: README.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 Weather, everybody wants to know how it is going to be during the week. Will it be rainy, windy, or sunny? Luckily for us, in the information age, there are open APIs to retrieve information about it.
 
+## Interfaces
+- **Atmosphere** (day and night)
+- **Clear** (day and night)
+- **Clounds** (day and night)
+- **Drizzle** (day and night)
+- **Rain** (day and night)
+- **Snow** (day and night)
+- **Thunderstorm** (day and night)
```
Comment (marceloamadeu) [inline README.md:11]: @pedrohsampaioo, I made a change in the other **README.md** (#4 )... I believe that after doing the PR we can update to update both.
Tks

**Review by marceloamadeu — COMMENTED** [review]
Some suggestions. Tks

File: app/lib/app/modules/home/home_page.dart (hunk 1 above)
Comment (pedrohsampaioo) [reply app/lib/app/modules/home/home_page.dart]: Hi, sure! 😆

File: README.md (hunk 2 above)
Comment (jacobaraujo7) [reply README.md:11]: @marceloamadeu I accepted your PR!
@pedrohsampaioo Could you get a new PR based on README.md from @marceloamadeu?

**Pull request closed without merging**